raclients/Cargo.toml

```diff
@@ -0,0 +1,14 @@
+[package]
+name = "raclients"
+version = "0.1.0"
+edition = "2021"
+
+[features]
+default = ["std", "frontend"]
+std = []
+alloc = []
```

Comment:
Please take a look at the old Cargo.toml, we need to enabled serde features depending on this features.

Reply:
What you did now is to use alloc every time, but it's not the common patter, please follow the old one where we enable alloc or std for serde create depending on our features.

Reply:
Yes, this needs to be fixed. I used alloc for everything as to not needing to deal with entangled dependencies. For a release, this would need to be changed.